Cross Region VM Restore Points.md

```diff
@@ -0,0 +1,396 @@
+            }
+         ],
+         "customData": "",
+         "windowsConfiguration": {
+            "provisionVMAgent": "true|false",
+            "winRM": {
+               "listeners": [
+                  {
+                     "protocol": "http"
+                  },
+                  {
+                     "protocol": "https",
+                     "certificateUrl": ""
+                  }
+               ]
+            },
+            "additionalUnattendContent": [
+               {
+                  "pass": "oobesystem",
+                  "component": "Microsoft-Windows-Shell-Setup",
+                  "settingName": "FirstLogonCommands|AutoLogon",
+                  "content": "<XML unattend content>"
+               }
+            ],
+            "enableAutomaticUpdates": "true|false"
+         },
+         "linuxConfiguration": {
+            "disablePasswordAuthentication": "true|false",
+            "ssh": {
+               "publicKeys": [
+                  {
+                     "path": "Path-Where-To-Place-Public-Key-On-VM",
+                     "keyData": "PEM-Encoded-public-key-file"
+                  }
+               ]
+            }
+         }
+      },
+      "storageProfile": {
+         "osDisk": {
+            "osType": "Windows|Linux",
+            "name": "OSDiskName",
+            "diskSizeGB": "10",
+            "caching": "ReadWrite",
+            "managedDisk": {
+               "id": "CSM Id of the managed disk",
+               "storageAccountType": "Standard_LRS"
+            },
+            "diskRestorePoint": {
+                   "id": "/subscriptions/<subId>/resourceGroups/<rgName>/restorePointCollections/<rpcName>/restorePoints/<rpName>/diskRestorePoints/<diskRestorePointName>",
+   }
+         },
+         "dataDisks": [
+            {
+               "lun": "0",
+               "name": "datadisk0",
+               "diskSizeGB": "10",
+               "caching": "ReadWrite",
+               "managedDisk": {
+                  "id": "CSM Id of the managed disk",
+                  "storageAccountType": "Standard_LRS"
+               },
+               "diskRestorePoint": {
+                   "id": "/subscriptions/<subId>/resourceGroups/<rgName>/restorePointCollections/<rpcName>/restorePoints/<rpName>/diskRestorePoints/<diskRestorePointName>",
+               }
+            }
+         ]
+      },
+      "diagnosticsProfile": {
+         "bootDiagnostics": {
+            "enabled": true,
+            "storageUri": " http://storageaccount.blob.core.windows.net/"
+         }
+      },
+   },
+   "provisioningState": "Succeeded | Failed | Creating | Deleting",
+   "provisioningDetails": {
+      "creationTime": "Creation Time of Restore point in UTC",
+   },
+   "instanceView": {
+      "statuses": [{
+          "code": "ReplicationState/succeeded",
+          "level": "Info",
+          "displayStatus": "Replication succeeded",
+          "message": "<will be populated in error scenarios>",
```

Comment:
drop "message" too -- since we don't get error details during failure scenarios today.